lib/shopify-cli/commands/deploy/script.rb

```diff
@@ -0,0 +1,76 @@
+require 'shopify_cli'
+
+module ShopifyCli
+  module Commands
+    class Deploy
+      class Script < ShopifyCli::Command
+        CMD_DESCRIPTION = "Deploy a script to the extension platform"
+        CMD_USAGE = "deploy <API Key>"
+
+        FAILED_TO_BUILD_MESSAGE = "Failed to build"
+        DEPLOY_SUCCEEDED_MSG = "{{v}} %{extension_point} script %{script_name} " \
+        "is deployed to app (API_KEY: {{green:%{api_key}}})"
+        BUILDING_MSG = "Building"
+        DEPLOYING_MSG = "Deploying"
+        BUILT_MSG = "Built"
+        DEPLOYED_MSG = "Deployed"
+
+        INVALID_EXTENSION_POINT = "Invalid extension point %{extension_point}"
+        SCRIPT_NOT_FOUND = "Could not find script %{script_name} for extension point %{extension_point}"
+
+        options do |parser, flags|
+          parser.on('--api_key=APIKEY') { |t| flags[:api_key] = t }
+          parser.on('--language=LANGUAGE') { |t| flags[:language] = t }
+        end
+
+        def call(args, _name)
+          form = Forms::DeployScript.ask(@ctx, args, options.flags)
+          return @ctx.puts(self.class.help) unless form
+
+          api_key = form.api_key
```

Comment:
Why are you re-assigning this. It is used twice? Just seems like extra code

Reply:
I didn't write this specific piece of code so I cant say for sure, but I see the argument both for and against this. It is more lines, but since its used in a couple different methods I would argue that it makes it easier to read the input